### PR TITLE
New Admin UI layout

### PR DESCRIPTION
**Pull request by hiveer**
**What this PR does**:
Renew admin panel UI layout:

image

**Which issue(s) this PR fixes**:

Fixes #

**Type of changes**
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

**Feature/Issue validation/testing**:

Please describe the tests that you ran to verify your changes and relevant result summary. 

- [ ] Test A
- [ ] Test B

- Logs

**Special notes for your reviewer**:

**Checklist**:

- [ ] I have added unit/e2e tests that prove your fix is effective or that this feature works.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have reviewed my own code and ensured that it follows the project's style guidelines.

**Release note**:

```release-note

```

### MR Summary:
*The summary is added by @codegpt.*

This Merge Request introduces a significant update to the Admin UI layout, focusing on enhancing the user experience and functionality. The changes include the addition of new Vue components for the admin interface, updates to existing components, and modifications to the backend to support these changes. Key updates include:

1. Implementation of a new admin layout with Vue components, including `AdminLayout`, `Container`, `Menu`, `Navbar`, `Pagination`, and `Table`.
2. Introduction of a new routing system (`/admin_next`) to accommodate the updated admin UI.
3. Backend adjustments to serve the new admin UI, including updates to handler and router configurations.
4. Styling enhancements and bug fixes in the frontend, including updates to Tailwind CSS configuration and Vue component styles.
5. Codebase cleanup and organization, including the removal of unused code and consolidation of admin components.

This MR aims to provide a more intuitive and efficient administrative interface, improving navigation, data presentation, and overall usability.

### CODE REVIEW

File: internal/handlers/render/admin.go [hunk 1]
```diff
@@ -34,3 +35,11 @@ func (i *AdminHandlerImpl) Index(ctx *gin.Context) {
 		i.renderBaseInstance.RenderTemplate(ctx, "admin_index", data)
 	}
 }
+
+func (i *AdminHandlerImpl) IndexNext(ctx *gin.Context) {
+	currentUser := ctx.MustGet("currentUser").(*models.User)
```
Comment (zhendi) [inline internal/handlers/render/admin.go:40]: @hiveer models should be import

File: internal/handlers/render/admin.go [hunk 2]
```diff
@@ -34,3 +35,11 @@ func (i *AdminHandlerImpl) Index(ctx *gin.Context) {
...
+	data := map[string]interface{}{
+		"roles": currentUser.Roles(),
+	}
+	renderTemplate(ctx, "admin_next", data)
```
Comment (zhendi) [inline internal/handlers/render/admin.go:44]: renderTemplate is undefined

File: internal/handlers/render/admin.go (hunk 1 above)
Comment (hiveer) [reply internal/handlers/render/admin.go]: done @zhendi

File: internal/handlers/render/admin.go (hunk 2 above)
Comment (hiveer) [reply internal/handlers/render/admin.go]: @zhendi done

**Review by zhendi — APPROVED** [review]
LGTM

**Pull request merged**